driver.php

```diff
@@ -203,14 +203,17 @@
 </p>
 <pre>
 <b>$</b> gcc -v hello.c
+COLLECT_GCC_OPTIONS=...
+ /usr/lib/gcc/aarch64-linux-gnu/11/cc1 hello.c ... -o /tmp/ccCKUy4c.s
 COLLECT_GCC_OPTIONS=...
  as -v -EL -mabi=lp64 -o /tmp/ccUkKMFH.o /tmp/ccCKUy4c.s
-COMPILER_PATH==...
+COMPILER_PATH=...
+LIBRARY_PATH=...
 COLLECT_GCC_OPTIONS=...
- /usr/lib/gcc/aarch64-linux-gnu/11/collect2 /tmp/ccIPliNQ.o
+ /usr/lib/gcc/aarch64-linux-gnu/11/collect2 ... /tmp/ccUkKMFH.o ...
 </pre>
 <p>
-It is a lot harder to read but we can still make out that an assembler step <code>as</code> took place, followed by a linking step via <code>collect2</code>. The call to the compiler via flag <code>-cc1</code> however is not featured.
+It is a lot harder to read but we can still make out that a compiler step <code>cc1</code> took place, followed by an assembler step <code>as</code>, followed by a linking step via <code>collect2</code>.
```

Comment:
This is a good. If you send a separate PR, I will merge it (I disagree with the change above).